README.md

```diff
@@ -1,19 +1,58 @@
-Check this [presentation](https://docs.google.com/presentation/d/1LnjIQT7tTrk7V7zK8xPE4LW-R5lJbAAPvDVEvPU6_xA/edit) for more information.  
+This tutorial shows some scenarios related to the Cilium and Spire integration.
+The image below represents the summary of the actions performed in each of them.
+
+<img src="imgs/background.png" alt="drawing" width="800"/>
 
 ## First steps
 
-- Download repository dependencies: `go vendor`
-- Setup cilium development environment
-- Checkout the branchs
-	- cilium: kinvolk:mauricio/spiffe-integration-poc-part2 
-	- cilium-proxy: kinvolk:mauricio/spiffe-integration-poc-part2 
-- Compile and install `cilium` and `cilium-proxy` 
-- Deploy `spire-agent` and `spire-server`: `kubectl -f spire.yaml`
+Download repository dependencies:
+
+```
+go vendor
+```
+
+Create minikube cluster:
+
+```
+minikube start --network-plugin=cni --memory=4096
+minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
+```
+
+Deploy manifest (cilium + spire + dependencies):
+
+```
+kubectl apply -f cilium-spire.yaml
+```
+
+Check pods status:
+
+```
+kubectl get pods -A
+NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE
+kube-system   cilium-74m7n                       1/1     Running   0          47s
+kube-system   cilium-operator-7c755f4594-2pk77   1/1     Running   0          3m25s
+kube-system   cilium-operator-b76f5d644-ccmtc    0/1     Pending   0          51s
```

Comment:
It´s strange, why 2 of 3 operator are with `Pending` status. The same happens in my env. to.

Reply:
Yes, same question here :man_shrugging: